lib/phantom.js

```diff
@@ -48,12 +48,14 @@ module.exports = function (b, opts) {
     var phantomPath = findPhantomJS();
     input  = through();
     output = phantomic(input, {
-      debug          : opts.debug,
-      port           : opts.port,
-      brout          : true,
-      phantomjs      : phantomPath,
-      'web-security' : opts['web-security'],
-      'ignore-ssl-errors': opts['ignore-ssl-errors']
+      debug              : opts.debug,
+      port               : opts.port,
+      brout              : true,
+      phantomjs          : phantomPath,
+      'web-security'     : opts['web-security'],
+      'ignore-ssl-errors': opts['ignore-ssl-errors'],
+      viewportWidth      : opts['viewport-width'],
```

Comment:
I did not respect the correct kebap-case everywhere when adding this to phantomic. We might want to fix that in there first, so we can get rid of that oddity in here.